core/app/models/workarea/catalog/customizations.rb

```diff
@@ -50,7 +50,13 @@ def initialize(product_id, attributes)
     @attributes = attributes.with_indifferent_access
 
     attributes.each do |name, value|
-      instance_variable_set("@#{name.to_s.titleize.systemize}", value)
+      ivar = if name.to_s.include?('_')
```

Comment:
Seems like we could just swap the `titleize` for `underscore` and achieve the same fix in a less brittle way? That `titleize` feels like it was the wrong idea in the first place.

Reply:
you're right, replacing it with underscore kept the tests passing.